WcaOnRails/app/controllers/users_controller.rb

```diff
@@ -69,13 +69,13 @@ def select_nearby_delegate
 
   def edit_avatar_thumbnail
     @user = user_to_edit
-    return if redirect_if_cannot_edit_user(@user)
+    return if redirect_unless_user(:can_change_users_avatar?, @user)
```

Comment:
I'm not used to the `and return` thing, but if it's lower mental load for a rubyist, then bring it on!